debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -649,6 +649,22 @@ public class OracleConnectorConfig extends HistorizedRelationalDatabaseConnector
             .withDescription("(Deprecated) if true, CONTINUOUS_MINE option will be added to the log mining session. " +
                     "This will manage log files switches seamlessly.");
 
+    public static final Field LOG_MINING_PATH_DICTIONARY = Field.create("log.mining.path.dictionary")
+            .withDisplayName("Defines the dictionary path for the mining session")
+            .withType(Type.STRING)
+            .withWidth(Width.LONG)
+            .withImportance(Importance.LOW)
+            .withValidation(OracleConnectorConfig::validateDictionaryFromFile)
+            .withDescription("Path to LogMiner dictionary file. This is required when the database is in" +
+                    " a different host than the Debezium connector");
```

Comment:
I don't believe the description here is technically accurate. The dictionary file is required because the connector is capturing changes from a read-only replica where the data dictionary details may not be available and where LogMiner is unable to write to the target database. Perhaps it makes more sense to simply say "this is required when using the connector against a read-only database replica."